client.c

```diff
@@ -222,7 +220,8 @@ void clientmgr (void *arg)
 
   UNUSED_ARG(arg);
 
-  /* Initialize invalid addresses. Remainder of members is already initialized to 0. */
+  /* Initialize invalid addresses */
+  memset(invalidAddresses, 0, sizeof(invalidAddresses));
```

Comment:
I think it's not needed because global and static variables are always initialized by zero.